internal/testrunner/reporters/console.go

```diff
@@ -0,0 +1,40 @@
+package reporters
+
+import (
+	"fmt"
+
+	"github.com/jedib0t/go-pretty/table"
+
+	"github.com/elastic/elastic-package/internal/testrunner"
+)
+
+func init() {
+	testrunner.RegisterReporter(ConsoleReporter, ReportConsole)
+}
+
+const (
+	// ConsoleReporter reports test results in a console-friendly tabular format
+	ConsoleReporter testrunner.TestReporter = "console"
+)
+
+// ReportConsole returns the given test results formatted in a console-friendly tabular format
+func ReportConsole(results []testrunner.TestResult) (string, error) {
```

Comment:
Should this function be exported?

Reply:
No, it doesn't need to be. Good catch, will fix.

Reply:
Fixed in dbc6931bfd641a1e9fa45fd0c271afcda4109e9c.